modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -205,9 +215,15 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     this.transaction.registry(this._registry);
     this.transaction.chainName(this._material.chainName);
     if (this._keyPair) {
-      this.transaction.sign(this._keyPair);
+      await this.transaction.sign(this._keyPair);
+    }
+    // TSS signature added
```

Comment:
This can be any signature created offline, not just from TSS

Reply:
This method was implemented as an async method so `await` was added, but looking back at that `transaction.sign`, there's nothing really async going on there. Maybe let's just remove the `async` modifier from `transaction.ts` and also the `@inheritDoc` too, doesn't seem to inherit any methods from the BaseTransaction

Reply:
Removed the await but it does inherit this method

[src/coin/dot/transactionBuilder.ts](https://github.com/BitGo/BitGoJS/blob/00cd5662bf9f89c9c4bdab948f6548107c9ef696/modules/account-lib/src/coin/baseCoin/baseTransactionBuilder.ts#L84-L90)

Reply:
Ah my bad, let me clarify, it is the `sign` method, doesnt see it in the `BaseTransaction` class: 

https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/src/coin/dot/transaction.ts#L46